app/views/eu_exit_business_readiness_requests/_request_details.html.erb

```diff
@@ -66,15 +66,15 @@
   :sector,
   as: :check_boxes,
   label: "What sector(s) is the content is about?",
-  collection: f.object.sector_options,
+  collection: Support::Requests::EuExitBusinessReadinessRequest::SECTOR_OPTIONS,
```

Comment:
This should be part of the first commit as it's related to sectors, not business activity.

Reply:
Fixed and fixed up. 😃

Reply:
LGTM! 👍